doc/tutorial/profiling.txt

```diff
@@ -33,11 +33,12 @@ functions using either of the following two options:
     - You can also combine the profile of many functions: 
     
       .. testcode::
+          :hide:
 
           profile = theano.compile.ProfileStats()
-          f = theano.function(..., profile=profile)
-          g = theano.function(..., profile=profile)
-          ...
+          f = theano.function(..., profile=profile)  # doctest: +SKIP
+          g = theano.function(..., profile=profile)  # doctest: +SKIP
+          ...  # doctest: +SKIP
```

Comment:
I changed the block type to "doctest" instead, which made "# doctest: + SKIP" work. The layout of "doctest" in html seems to be the same to "testcode."